locale/en/blog/announcements/interactive-2015-programming.md

```diff
@@ -0,0 +1,60 @@
+layout: blog-post.hbs
+---
+
+> Inaugural Conference to Advance the Use of Node.js Within Backend, Frontend, IoT Applications
+
+SAN FRANCISCO, Oct. 20, 2015 – [The Node.js Foundation](https://nodejs.org/en/foundation/), a community-led and industry-backed consortium to advance the development of the Node.js platform, today announced initial programming for [Node.js Interactive](http://events.linuxfoundation.org/events/node-interactive). This inaugural event, which is being led by the newly formed Node.js Foundation in cooperation with the Linux Foundation, will be held December 8-9, 2015, in Portland, Ore.
+
+Node.js has become ubiquitous in almost every ecosystem in technology and is consistently being used more in mainstream enterprises. To continue to evolve the platform, Node.js Interactive brings together a wide range of community, projects, products and companies to create an educational and collaborative space. With more than 700 attendees expected, Node.js Interactive will provide a way to network with other developers and engineers within this diverse community.  
+
+Node.js Interactive will also focus on three tracks: Frontend, Backend and the Internet of Things (IoT); talks for each track were selected in collaboration with track chairs [Jessica Lord](https://github.com/jlord/) (Frontend), [C J Silvero](https://github.com/ceejbot) (Backend) and [Kassandra Perch](https://github.com/nodebotanist) (IoT). A few highlights include:
+
+Frontend Session Highlights:
+* JavaScript, For Science! *with* Max Ogden, Computer Programmer for Dat Project
+* Making Your Node.js Applications Debuggable *with* Patrick Mueller, Senior Node Engineer at NodeSource
+* Node Intl: Where We Are, What's Next *with* Steven Loomis, Senior Software Engineer at IBM
+* Rapid Development of Data Mining Applications in Node.js *with* Blaz Fortuna, Research Consultant for Bloomberg L.P., Senior Researcher at Jožef Stefan Institute and Partner at Quintelligence
+* Real-Time Collaboration Sync Strategies *with* Todd Kennedy, CTO of Scripto
+* Rebuilding the Ship as It Sails: Making Large Legacy Sites Responsive *with* Philip James, Senior Software Engineer at Eventbrite
+
+Backend Session Highlights:
+* Building and Engaging High-Performance Teams in the Node.js Ecosystem *with* Chanda Dharap, Director of Engineering at StrongLoop, an IBM company
+* Microservice Developer Experience *with* Peter Elger, Director of Engineering at nearForm
+* Modernizing Winston for Node.js v4 *with* Charlie Robbins, Director of Engineering UX Platform at GoDaddy
+* Node.js API Pitfalls, Can You Spot Them? *with* Sam Roberts, Node/Ops Developer at StrongLoop, an IBM Company
+* Node.js Performance Optimization Case Study *with* Bryce Baril, Senior Node Engineer at NodeSource
+* Resource Management in Node.js *with* Bradley Meck, Software Engineer at NodeSource
+
+IoT Session Highlights:
+* Contributing to Node Core *with* Jeremiah Senkpiel, Node.js Web Developer at Zambah Inc
```

Comment:
I most definitely no longer work for Zambah, where did you find that? _linkedin_? :P